gulpfile.js

```diff
@@ -327,18 +336,39 @@ gulp.task('debug', ['dist', 'clean-debug'], function (done) {
                 process.exit(1);
             });
         }
-        var exec = require('child_process').exec;    
-        if (platforms.length === 1) {
-            var run = getRunDebugAppCommand(platforms[0]);
-            console.log('Starting debug app (' + run + ')...');
-            exec(run);
-        } else {
-            console.log('More than one platform specified, not starting debug app');
-        }        
-        done();
+        // Execute post build task
+        runSequence('post-build', function() {
+            var exec = require('child_process').exec;    
+            if (platforms.length === 1) {
+                var run = getRunDebugAppCommand(platforms[0]);
+                console.log('Starting debug app (' + run + ')...');
+                exec(run);
+            } else {
+                console.log('More than one platform specified, not starting debug app');
+            }        
+            done();
+        });
     });
 });
 
+gulp.task('post-build', function(done) {
```

Comment:
Can the post-build function be called from the command line? If not, what is the benefit of create it as a gulp task and not as a simple function? Making it a gulp task, makes the code difficult to read, because it depends of a new global var `destDir`. 
I don't know if it's only me, but I don't like the global variables, I prefer convert this task to a function and pass the var as parameter. It makes the code easier to read. The same with `platforms`, if this task is converted into a function, it can be passed as parameter and not let it as global var.

Reply:
Yes, you are right. There is no logical reason to execute `post-build` from the commandline. I will convert this to a regular function instead of a gulp task.